Assets/MRTK/Core/Utilities/StandardShader/ClippingPrimitive.cs

```diff
@@ -339,18 +397,14 @@ protected virtual void UpdateRenderers()
 
         protected void ToggleClippingFeature(bool keywordOn)
         {
-            if (renderers != null)
-            {
-                for (var i = 0; i < renderers.Count; ++i)
+            if (renderers.Count > 0)
```

Comment:
nit, but this `if` looks like it's missing its braces

Reply:
Fixed!